boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -399,4 +409,147 @@ TEST_F(
     }
 }
 
+// DCM-8
+TEST_F(
+    DcmStateMachineTest,
+    regen_allowed_only_when_going_faster_than_5kph_and_airs_closed)
+{
+    SetInitialState(App_GetDriveState());
+
+    // Turn the DIM start switch on to prevent state transitions in
+    // the drive state.
+    EXPECT_TRUE(App_BuzzerSignals_IsOn(world));
+    App_CanRx_DIM_SWITCHES_SetSignal_START_SWITCH(
+        can_rx_interface, CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE);
+    ASSERT_EQ(
+        App_GetDriveState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
+
+    // Press down accelerator pedal and regen paddle
+    App_CanRx_FSM_PEDAL_POSITION_SetSignal_MAPPED_PEDAL_PERCENTAGE(
+        can_rx_interface, 60.0f);
+    App_CanRx_DIM_REGEN_PADDLE_SetSignal_MAPPED_PADDLE_POSITION(
+        can_rx_interface, 50.0f);
+
+    // Close AIRs and assert regen is still off
+    App_CanRx_BMS_AIR_STATES_SetSignal_AIR_NEGATIVE(
+        can_rx_interface, CANMSGS_BMS_AIR_STATES_AIR_NEGATIVE_CLOSED_CHOICE);
+    App_CanRx_BMS_AIR_STATES_SetSignal_AIR_POSITIVE(
+        can_rx_interface, CANMSGS_BMS_AIR_STATES_AIR_POSITIVE_CLOSED_CHOICE);
+    LetTimePass(state_machine, 10);
+    EXPECT_TRUE(
+        App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx_interface) >= 0);
```

Comment:
Why can't we compare the expected value with the actual torque request here?

Reply:
The idea was that the current torque request calculations are somewhat of a placeholder. Eventually torque requests will be scaled by the PID controller so it won't be easy to do an EXPECT_EQ directly, but I guess for this PR it would be better to just check them directly